indico/modules/events/timetable/legacy.py

```diff
@@ -33,7 +33,9 @@ class TimetableSerializer(object):
     def __init__(self, management=False):
         self.management = management
 
-    def serialize_timetable(self, event, days=None, hide_weekends=False, strip_empty_days=False):
+    def serialize_timetable(self, event, days=None, hide_weekends=False, strip_empty_days=False, tzinfo=None):
+        if not tzinfo:
```

Comment:
Hmm if we need to use it for offline site generation it indeed makes sense - in that case we want the event tz for the display page (even though it'll probably always match the display tz due to the lack of a user/session)